<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.1 (Python 3.8.10 on win32)" generated="20220509 16:22:01.509" rpa="false" schemaversion="2">
<suite id="s1" name="Principalbd" source="c:\Users\HP\Downloads\Introduccion_RobotFramework\principalbd.robot">
<kw name="Connect To Database" type="SETUP">
<arg>pymysql</arg>
<arg>${DBNAME}</arg>
<arg>${DBUSER}</arg>
<arg>${DBPASS}</arg>
<arg>${DBHOST}</arg>
<arg>${DBPORT}</arg>
<msg timestamp="20220509 16:22:01.708" level="FAIL">No keyword with name 'Connect To Database' found.</msg>
<status status="FAIL" starttime="20220509 16:22:01.703" endtime="20220509 16:22:01.770"/>
</kw>
<test id="s1-t1" name="Ingresar datos a la base datos">
<status status="FAIL" starttime="20220509 16:22:01.770" endtime="20220509 16:22:01.770">Parent suite setup failed:
No keyword with name 'Connect To Database' found.</status>
</test>
<doc>Es un test de pruebas a la base de datos bdEjemplo

Palabras reservadas para importar archivos</doc>
<status status="FAIL" starttime="20220509 16:22:01.564" endtime="20220509 16:22:01.775">Suite setup failed:
No keyword with name 'Connect To Database' found.</status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Principalbd">Principalbd</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20220509 16:22:01.630" level="ERROR">Error in file 'c:\Users\HP\Downloads\Introduccion_RobotFramework\sourcebd.robot' on line 2: Importing library 'DatabaseLibrary' failed: ModuleNotFoundError: No module named 'DatabaseLibrary'
Traceback (most recent call last):
  None
PYTHONPATH:
  c:\Users\HP\.vscode\extensions\robocorp.robotframework-lsp-0.46.0\src\robotframework_debug_adapter
  C:\Users\HP\Downloads\Introduccion_RobotFramework
  c:\Users\HP\.vscode\extensions\robocorp.robotframework-lsp-0.46.0\src
  c:\Users\HP\.vscode\extensions\robocorp.robotframework-lsp-0.46.0\src\robotframework_ls\vendored
  C:\Users\HP\AppData\Local\robocorp\holotree\b1f3c244e_da28c852\python38.zip
  C:\Users\HP\AppData\Local\robocorp\holotree\b1f3c244e_da28c852\DLLs
  C:\Users\HP\AppData\Local\robocorp\holotree\b1f3c244e_da28c852\lib
  C:\Users\HP\AppData\Local\robocorp\holotree\b1f3c244e_da28c852
  C:\Users\HP\AppData\Local\robocorp\holotree\b1f3c244e_da28c852\lib\site-packages
  C:\Users\HP\AppData\Local\robocorp\holotree\b1f3c244e_da28c852\lib\site-packages\win32
  C:\Users\HP\AppData\Local\robocorp\holotree\b1f3c244e_da28c852\lib\site-packages\win32\lib
  C:\Users\HP\AppData\Local\robocorp\holotree\b1f3c244e_da28c852\lib\site-packages\Pythonwin</msg>
</errors>
</robot>
